packages/react-components/react-calendar-compat/library/src/components/CalendarMonth/CalendarMonth.tsx

```diff
@@ -327,7 +327,7 @@ function getYearStrings({
       rangeAriaLabel: yearRangeToString,
       prevRangeAriaLabel: yearRangeToPrevDecadeLabel,
       nextRangeAriaLabel: yearRangeToNextDecadeLabel,
-      headerAriaLabelFormatString: strings.yearPickerHeaderAriaLabel,
+      headerAriaLabelFormatString: strings.monthPickerHeaderAriaLabel,
```

Comment:
MonthHeader is misleading for year selection but this is because of those lines in `dateFormatting.defaults`:
```ts
  monthPickerHeaderAriaLabel: '{0}, change year',
  yearPickerHeaderAriaLabel: '{0}, change month',
```
And those strings are exported so I shouldn't change them. There's a way to add a new string, but I don't think it's a good solution so I just used existing one.